x-pack/plugin/sql/qa/src/main/resources/datetime.csv-spec

```diff
@@ -621,6 +682,15 @@ null    | 1999-04-30 00:00:00.000Z
 F       | 1997-05-19 00:00:00.000Z
 ;
 
+dateTruncHavingWithInterval-Ignore
```

Comment:
Please add a comment that references the issue you have opened similar to: https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/sql/qa/src/main/resources/examples/example.csv-spec#L64

Reply:
fixed